src/error-snippets/frames.ts

```diff
@@ -0,0 +1,61 @@
+import _ from "lodash";
+import ErrorStackParser from "error-stack-parser";
+import logger from "../utils/logger";
+import { softFileURLToPath } from "./utils";
+import type { ResolvedFrame, SufficientStackFrame } from "./types";
+
+// prettier-ignore
+const frameRelevance: Record<string, { value: number; matcher: (fileName: string) => boolean }> = {
```

Comment:
https://nodejs.org/api/errors.html#errorstack

> The location information will be one of:

- `native`, if the frame represents a call internal to V8 (as in [].forEach).
- `plain-filename.js:line:column`, if the frame represents a call internal to Node.js.
- `/absolute/path/to/file.js:line:column`, if the frame represents a call in a user program (using CommonJS module system), or its dependencies.
- `<transport-protocol>:///url/to/module/file.mjs:line:column`, if the frame represents a call in a user program (using ES module system), or its dependencies.

Reply:
Ranking them as:
- 0: can't extract code snippet; useless
- 1: better than nothing
- 2: better than wdio internals, but worse, than user code part (these could be testplane parts)
- 3: user code. If frame of user code appears, always should pick it